tests/Oryx.BuildImage.Tests/DotNetCore/DotNetCoreSampleAppsTest.cs

```diff
@@ -29,94 +29,14 @@ public DotNetCoreSampleAppsTest(ITestOutputHelper output) : base(output)
 
         private readonly string SdkVersionMessageFormat = "Using .NET Core SDK Version: {0}";
 
-        [Fact]
-        public void Builds_NetCore10App_UsingNetCore11_DotNetSdkVersion()
```

Comment:
We need to have coverage for this version in our platform so can you check if there is already a test for this version? else we can change this test to use the github-actions image. Similarly can you check for other tests too?

Reply:
Sure, will check the coverage for this version.

Reply:
I just found that dynamic install is currently failing for 1.0 and 1.1 versions in github-actions and lts-versions image. I think it is because these two images are missing pre-requisite system packages for those versions to work.
For example, you can add the following highlighted system packages:

![image](https://user-images.githubusercontent.com/6559784/92818390-828d1600-f37c-11ea-8b14-eafb71cf28a3.png)